game/src/js/scenes/DungeonScene.js

```diff
@@ -69,6 +69,7 @@ export default class DungeonScene extends BaseScene {
     this.tileset1 = this.map1.addTilesetImage('stone-tiles', 'stone-tiles');
     this.layer1 = this.map1.createStaticLayer('Dungeon Map', this.tileset1, -1000, -600);
     this.collisionLayer = this.map1.createStaticLayer('Collision', this.tileset1, -1000, -600);
+    this.layer1.setCollisionByExclusion([5,0], true);
```

Comment:
Actually if the collision is applied this way we don't need a collision layer on the map. What this is basically doing is excluding tiles 5 and 0 from the list of colliding tiles.

Reply:
Oh sure. If the collision layer is not really the collision layer it can get confusing down the line. Right now it makes sense for us, but for someone else looking at the code it will not be obvious.